.github/workflows/test.yaml

```diff
@@ -0,0 +1,140 @@
+name: Test
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        description: 'Environment to run the tests in'
+        required: false
+        default: null
+        type: string
+      python-version:
+        description: 'Python version to run the tests in'
+        required: true
+        type: string
+      use-llms:
+        description: 'Use LLM in the tests'
+        required: true
+        type: string
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    environment: ${{ inputs.environment }}
+    services:
+      nats:
+        image: diementros/nats:js
+        ports:
+          - 4222:4222
+      postgres-wasp:
```

Comment:
We don't need postgres, nats in workflow to test core fastagency python library.

@davorrunje Shall I remove it now itself or later when we remove the unnecessary code from fastagency/ directory?